### PR TITLE
Next

### PR DESCRIPTION
**Pull request by alexjoverm**
Same than #10 for Vue 3

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,37 +1,105 @@
...
+
+Check out the [playground](/../../tree/next/playground) for a full example.
+
+### Compatibility
+
+This plugin is for Vue 3. Thus, it supports the [same browsers as Vue 3](https://github.com/vuejs/rfcs/blob/master/active-rfcs/0038-vue3-ie11-support.md). In short: all modern browsers, dropping IE support.
+
+## 🔗 Related Links
+
+- **[Add a headless CMS to Vue.js in 5 minutes](https://www.storyblok.com/tp/add-a-headless-CMS-to-vuejs-in-5-minutes?utm_source=github.com&utm_medium=readme&utm_campaign=storyblok-vue)**: Quick-start guide on getting up and running with Storyblok and Vue.
```
Comment (samuells) [inline README.md:87]: @alexjoverm make it sense to link this here? Shouldn't we create Vue3 article? 

I would also mention Storyblok Vue/Nuxt Technology Hub.

File: package.json [hunk 2]
```diff
@@ -1,20 +1,55 @@
...
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/storyblok/storyblok-vue"
-  },
-  "keywords": [
-    "vue",
-    "storyblok"
+  "name": "@storyblok/vue-ws",
+  "private": true,
+  "version": "0.0.1",
```
Comment (samuells) [inline package.json:4]: @alexjoverm has any value this version here?

**Review by samuells — APPROVED** [review]
In general, approved from my side. Pls check the questions - same as in #10 ;)

File: package.json (hunk 2 above)
Comment (alexjoverm) [reply package.json:4]: Not at all, it's the workspace root so it just needs a name and a version to work

File: README.md (hunk 1 above)
Comment (alexjoverm) [reply README.md:87]: Very good points!

**Pull request merged**